src/Asset.js

```diff
@@ -73,7 +72,7 @@ class Asset {
 
     let resolved = path
       .resolve(path.dirname(from), url)
-      .replace(/[\?#].*$/, '');
+      .replace(/[?#].*$/, '');
```

Comment:
I'm not very good with regex, so simply ignore if I'm being stupid here, but looks like ESLint removed `\`, which could potentially change the result of `resolved`. Wouldn't that mean that it would now also remove `?` and `#` from the path? 🤔

Reply:
In this case, they are the same. 

```
[\?#]
=> match any item in the list: 
- literal ?
- literal #

[?#]
=> match any item in the list:
`` ?#

Reply:
Yeah @brandon93s I'm the type that has too look up SO for the simplest regex's. 😆 but thanks for looking into it!

Reply:
Yep: ESLint was complaining about unecessary escapes.